common/src/tracer.rs

```diff
@@ -0,0 +1,158 @@
+use opentelemetry::{
+    propagation::{Extractor, Injector},
+    trace::{TraceContextExt, TraceError},
+};
+use opentelemetry_otlp::WithExportConfig;
+use rdkafka::message::{BorrowedHeaders, Headers, OwnedHeaders};
+use std::fmt::Debug;
+use tracing::Span;
+use tracing_opentelemetry::{self, OpenTelemetrySpanExt};
+use tracing_subscriber::layer::SubscriberExt;
+
+const ENDPOINT: &str = "http://localhost:4317/v1/traces";
```

Comment:
I would rather this default be handled in command line parsing of the tool that uses OpenTelemetry, this makes it more explicit on the CLI and reduces boilerplate code.